### PR TITLE
refactor: update InitGenesis and ExportGenesis

### PR DESCRIPTION
**Pull request by lumtis**
# Description

Update the export and import genesis

When merged we can make a cherry pick to make the change on `develop`

I'm creating the branch now for availability. I think we could also update Validate methods and tests

### CODE REVIEW

File: x/emissions/genesis.go [hunk 1]
```diff
@@ -10,15 +10,17 @@ import (
 // state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, ak types.AccountKeeper, genState types.GenesisState) {
 	k.SetParams(ctx, genState.Params)
+
+	// This create module accounts if not created
+	// TODO: find a proper way  to create module accounts?
```
Comment (kingpinXD) [inline x/emissions/genesis.go:15]: We can add methods for withdrawable emissions . That would need a change in proto files though

**Review by kingpinXD — APPROVED** [review]

File: x/emissions/genesis.go (hunk 1 above)
Comment (lumtis) [reply x/emissions/genesis.go:15]: Yes, does it have a logical on-chain usage right now? It seems right now it is only used for documentation purposes.
But it should be straightforward to add.

Maybe we could try to maintain somewhere a TODO list for when adding new state components so we always add the value in the genesis in the same PR we add the type.

**Comment by lumtis** [comment]
Let me know if we can confirm we don't need this to the hotfix branch, I will port it to develop instead

**Comment by lumtis** [comment]
Recreated the branch for `develop` since no longer needed for the hotfix https://github.com/zeta-chain/node/pull/1023

**Pull request closed without merging**